src/app/pages/homepage/homepage.ts

```diff
@@ -0,0 +1,44 @@
+import { Component} from '@angular/core';
+
+@Component({
+  selector: 'app-homepage',
+  templateUrl: 'homepage.html',
+  styleUrls: ['homepage.scss']
+})
+export class Homepage {
+  private _imagePath: string = '../assets/img/homepage/';
+
+  homePageContent = [
```

Comment:
My repo is no longer forked from this repo after it changed to public. Let me merge this and do it in another pr